python/datasources/data_sources.py

```diff
@@ -29,5 +30,6 @@
     StateNames.get_id(): StateNames(),
     UrgentCareFacilities.get_id(): UrgentCareFacilities(),
     ACSHealthInsurance.get_id(): ACSHealthInsurance(),
-    ACSHouseholdIncomeDatasource.get_id(): ACSHouseholdIncomeDatasource()
+    ACSIncomeDatasource.get_id(): ACSIncomeDatasource()
```

Comment:
I don't think you mean to revert ACSHouseholdIncomeDatasource -> ACSIncomeDatasource on this line